calico/_includes/content/install-openshift-manifests.md

```diff
@@ -1,6 +1,7 @@
 Download the {{site.prodname}} manifests for OpenShift and add them to the generated manifests directory:
 
 ```bash
-wget -qO- {{ "/manifests/ocp.tgz" | absolute_url }} | tar xvz -C calico
+mkdir calico
+wget -qO- {{ "/manifests/ocp.tgz" | absolute_url }} | tar xvz --strip-components=1 -C calico
```

Comment:
Do we even need the intermediate `calico` dir? I suppose it doesn't hurt, but we could just go straight into `manifests`?

Reply:
It's not needed but I liked the original intention of having the Calico manifests in a separate place and copying them over so I left it like that.